app/views/api/v3/analytics/user_analytics/_monthly_gender_table.html.erb

```diff
@@ -0,0 +1,49 @@
+<!-- the "all" table -->
+<table class="progress-table <%= data_table_name %> hypertension:all">
+  <tr class="total">
+    <td class="row-label">All-time</td>
+    <td>
+      <%= zero_if_unavailable(@statistics
+                                  .dig(:all_time, :grouped_by_gender, resource)
+                                  .values
+                                  .inject(:+)) %>
```

Comment:
`.sum`? PS, this sort of logic can also be extracted to a presenter, keeping the views very simple:

`<%= all_time_count %>` or whatever.

Reply:
I have made some adjustments to this. More consolidated notes coming up.